module-2/lab-subsetting-and-descriptive-stats/your-code/main.ipynb

```diff
@@ -631,13 +631,6 @@
     "temperatureMS(temp)"
```

Comment:
You can use in your groupby questions the agg function to calculate basic statistics (sum, mean....) by group. 
You should comment the results of  boxplot diagram. 
In challenge 3 , you can skip the first column of orders csv file when reading it with pandas. That column is useless and can cause misunderstanding. 
One way of applying lambda 
```Python
employee.groupby(['Department']).agg({'Salary':lambda x: np.max(x)-np.min(x)})
```
You have a hint here but you haven't used this function. However, your code works. I include below another way of doing this stuff
```Python 
#How many customers do we have in Spain?
#Hint : Use value_counts()
orders.loc[orders['Country']=='Spain']['Country'].value_counts()
```